### PR TITLE
don't wrap the category label

### PR DESCRIPTION
**Pull request by mrblippy**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [x] screenshots are included showing significant UI changes

##### Description of change

#1306
 before
before

after
less unsightly

[contributor license agreement]: https://www.apereo.org/node/676
[commit guidelines]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by PenghaiZhang** [comment]
I feel it's better to make these numbers aligned.

**Comment by edalex-ian** [comment]
Yeah I'm not sure on this one either. Visually now it's pretty confusing. I wonder if instead of:

```typescript
    <Grid container spacing={1}>
      <Grid item>
        <Typography>{category}</Typography>
      </Grid>
      <Grid item>
        <Typography color="textSecondary">{`(${count})`}</Typography>
      </Grid>
    </Grid>
```

All we need is:

```typescript
    <>
        <Typography>{category}</Typography>
        <Typography color="textSecondary">{` (${count})`}</Typography>
    </>
```

(Note, I also attempt to sneak in a space before the `(${count})`.)

Another option do we want to consider fixing the count to the far right top - kind of the mirror of the checkbox. Maybe we should discuss as a group tomorrow, and drag in @cathfitz (seeing she raised the initial issue).

**Comment by mrblippy** [comment]
Latest update, with checkboxes aligned to top, and counts aligned to top-right. It required more custom styling than i would have preferred. 

aligned to top

Another alternative would be to replace the FormControlLabel with a three column grid, which yields very similar results.

**Comment by mrblippy** [comment]
Well damn, it appears that the test data i used was kind of crummy. And masked some extreme dodginess

**Comment by mrblippy** [comment]
Ok, I have rejigged this again. Now using better data. This tuned out to be even fiddlier than i imagined. I think this is the best i'm going to get it :( 
@edalex-ian it should be good to review now
with better data

**Comment by edalex-ian** [comment]
Heya @mrblippy 

As discussed, I reckon let's just keep this simple. The new changes with the checkbox is great, but for the count let's just tag it onto the end of the text for the category with secondary text styling. :wink:

**Comment by mrblippy** [comment]
@edalex-ian yeah, that's what i have ended up doing. have another look at your leisure

File: Source/Plugins/Core/com.equella.core/js/tsrc/search/components/CategorySelector.tsx [hunk 1]
```diff
@@ -158,14 +167,13 @@ export const CategorySelector = ({
...
-        <Typography>{category}</Typography>
-      </Grid>
-      <Grid item>
-        <Typography color="textSecondary">{`(${count})`}</Typography>
-      </Grid>
-    </Grid>
+    <>
+      <Typography display="inline">{category}</Typography>
+      <Typography color="textSecondary" display="inline">
+        {" "}
```
Comment (edalex-ian) [inline Source/Plugins/Core/com.equella.core/js/tsrc/search/components/CategorySelector.tsx:173]: I understand you're address this. Good plan. :+1:

File: Source/Plugins/Core/com.equella.core/js/tsrc/search/components/CategorySelector.tsx [hunk 2]
```diff
@@ -271,13 +289,12 @@ export const CategorySelector = ({
         <ListItem divider key={id}>
           <Grid container direction="column">
             <Grid item>
-              <Typography variant="subtitle1">{name}</Typography>
+              <Typography variant="subtitle1" style={{ fontWeight: 500 }}>
```
Comment (edalex-ian) [inline Source/Plugins/Core/com.equella.core/js/tsrc/search/components/CategorySelector.tsx:292]: hmmm, that style is an interesting one. You couldn't just wrap `{name}` in `<strong>`?

**Review by edalex-ian — APPROVED** [review]
Good stuff dude. Once you've made your tweaks and got a green build. Merge on. :+1:

**Pull request merged**